core/src/main/java/com/linecorp/armeria/internal/common/Http2ObjectEncoder.java

```diff
@@ -82,8 +82,14 @@ public final ChannelFuture doWriteData(int id, int streamId, HttpData data, bool
     }
 
     @Override
-    public final ChannelFuture doWriteReset(int id, int streamId, Http2Error error) {
+    public final ChannelFuture doWriteReset(int id, int streamId, Http2Error error,
+                                            boolean sendResetIfRemoteIsOpen) {
         final Http2Stream stream = encoder.connection().stream(streamId);
+
+        if (sendResetIfRemoteIsOpen && !stream.state().localSideOpen()) {
```

Comment:
Q: Why is the local side open checked? `sendResetIfRemoteIsOpen` says reset if the remote is open.

Reply:
I copied it from [here](https://github.com/line/armeria/blob/b2e31ee12941a7dd3883dde3a66f3ececa94e8fb/core/src/main/java/com/linecorp/armeria/server/ServerHttp2ObjectEncoder.java#L135-L137) and I think it's also fixed. 😓